pom.xml

```diff
@@ -194,6 +195,11 @@
                 <artifactId>commons-text</artifactId>
                 <version>${commons-text.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.apache.commons</groupId>
+                <artifactId>commons-lang3</artifactId>
```

Comment:
Out of curiosity: What is this used for?

Reply:
`StringUtils.normalizeSpace` when logging.